assets/dev/js/editor/document/repeater/commands/select.js

```diff
@@ -0,0 +1,64 @@
+import CommandHistory from 'elementor-document/commands/base/command-history';
+
+/**
+ * Command used to select current working repeater item.
+ */
+export class Select extends CommandHistory {
+	static restore( historyItem, isRedo ) {
+		const container = historyItem.get( 'container' ),
+			data = historyItem.get( 'data' );
+
+		$e.run( 'document/repeater/select', {
+			container,
+			index: isRedo ? data.current : data.prev,
+		} );
+	}
+
+	validateArgs( args ) {
+		this.requireContainer( args );
+		this.requireArgumentType( 'index', 'number', args );
+
+		const { containers = [ args.container ] } = args;
+
+		// When there multiple containers, then its not supported.
+		if ( containers.length > 1 ) {
+			throw new Error( 'Multiple containers are not supported.' );
+		}
+	}
+
+	getHistory( args ) {
+		const { container, index } = args,
+			editSettings = container.model.get( 'editSettings' ),
+			current = editSettings.get( 'activeItemIndex' ) || 1;
```

Comment:
Shouldn't it be `??` (for 0 values)?
```suggestion
			current = editSettings.get( 'activeItemIndex' ) ?? 1;
```

Reply:
It starts from 1.